docs/contributing/contributing.md

```diff
@@ -19,6 +19,10 @@ Contributions can be either officially supported (by Palo Alto Networks, a Techn
 
 For more information about the different support levels, check the [official documentation](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-2/cortex-xsoar-admin/marketplace/content-packs-support-types.html).
 
+## What to contribute
+If you’re looking for an easy way to start contributing Content to Cortex XSOAR, have a look at the issues labeled as [good first issue](https://github.com/demisto/content/labels/good%20first%20issue), which are small issues that would require easy patches.
+
+These are issues that we believe are particularly well suited for external contributions. If you decide to start on an issue, assign yourself so that other people know that you're working on it.
```

Comment:
```suggestion
If you’re looking for an easy way to start contributing Content to Cortex XSOAR Marketplace, have a look at the issues labeled as [good first issue](https://github.com/demisto/content/labels/good%20first%20issue), which are low-hanging fruits.

These issues are particularly well suited for external contributions. If you decide to start on an issue, [assign yourself](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users#assigning-an-individual-issue-or-pull-request) so other contributors from the community know that you're working on it.

In case you are in need of assistance, feel free to reach out to us via [DFIR](https://start.paloaltonetworks.com/join-our-slack-community), at the #demisto-developers channel.
```

Reply:
@bakatzir we already have reference to the dfir in contributor guidelines, do you think we also need it here?

Reply:
Not really. agreed.